windows/windows_deps_headless_installer.py

```diff
@@ -52,6 +52,8 @@
         'opencv': 'https://windows-post-install.s3-us-west-2.amazonaws.com/opencv-windows-4.1.2-vc14_vc15.zip',
         'cudnn': 'https://windows-post-install.s3-us-west-2.amazonaws.com/cudnn-9.2-windows10-x64-v7.4.2.24.zip',
         'nvdriver': 'https://windows-post-install.s3-us-west-2.amazonaws.com/nvidia_display_drivers_398.75_server2016.zip',
+        'perl': 'http://strawberryperl.com/download/5.30.1.1/strawberry-perl-5.30.1.1-64bit.msi',
+        'ninja': 'https://github.com/ninja-build/ninja/releases/download/v1.10.0/ninja-win.zip',
```

Comment:
ninja is installed here: 

https://github.com/aiengines/ci/blob/master/windows/setup.ps1#L46

Reply:
removed